SeniorSlideshowProject/src/SlideshowMaker.java

```diff
@@ -24,12 +24,20 @@
 import javax.swing.JLabel;
 import java.awt.Color;
 import javax.swing.JRadioButton;
+import javax.swing.SwingConstants;
+import javax.swing.AbstractListModel;
+import java.awt.Font;
+import javax.swing.ImageIcon;
+import java.awt.event.MouseAdapter;
+import java.awt.event.MouseEvent;
 
 public class SlideshowMaker extends JFrame {
 
 	private JPanel MainPanel;
 	private JTextField LayoutOfThe;
-
+	private JLabel lblImagepreview = new JLabel("");

```

Comment:
Starting here are some variables I moved to be class instead of local to modify for event handlers. contextSwitch will ultimately need to be removed, just wanted an easy binary variable for switching between images as a preview.